contao/dca/tl_module.php

```diff
@@ -356,6 +357,12 @@
                 'includeBlankOption' => true
             ),
             'sql'              => "varchar(64) NOT NULL default ''"
-        )
+        ),
+		'guests' => array
```

Comment:
Defining 'guests' with a new array will truncate the existing array and dca for 'guests'